modin/experimental/cloud/rayscale.py

```diff
@@ -136,23 +136,30 @@ def __make_config(self):
 
         # NOTE: setup_commands may be list with several sets of shell commands
         # this change only first set defining the remote environment
-        res = self._update_conda_requirements(config["setup_commands"][0])
+        res = self._update_conda_requirements(
+            config["setup_commands"][0], self.add_conda_packages
+        )
         config["setup_commands"][0] = res
 
         return _bootstrap_config(config)
 
     @classmethod
-    def _conda_requirements(cls):
+    def _conda_requirements(cls, add_conda_packages: list = None):
         reqs = []
 
         reqs.append(f"python=={cls._get_python_version()}")
 
+        if add_conda_packages:
+            reqs.extend(add_conda_packages)
+
         return reqs
 
     @classmethod
-    def _update_conda_requirements(cls, setup_commands: str):
+    def _update_conda_requirements(
+        cls, setup_commands: str, add_conda_packages: list = None
```

Comment:
same, just make it a regular method (and you won't have to pass `add_conda_packages` around)